test/bluemix.test.js

```diff
@@ -134,6 +138,43 @@ describe('loopback:bluemix generator', function() {
       done();
     });
   });
+
+  it('should login with user/password', function(done) {
+    if (!process.env.BLUEMIX_EMAIL) {
+      console.error('Missing BLUEMIX_EMAIL and BLUEMIX_PASSWORD env vars');
+      return done();
```

Comment:
This makes the test pass anyway, will fix it. Applies to 'should login with SSO passcode' too.